### PR TITLE
feat: Added python debugging script for scale sets

### PR DESCRIPTION
**Pull request by rhartnett-zscaler**
Information about how to run the script is included in the README that is apart of this PR.

### CODE REVIEW

File: scripts/support_bundle/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+cd scripts/support_bundle
+python -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+python generate_support_bundle.py --resource_group <resource-group> --time_duration <time-in-min> --subscription_id <subscription-id> --tenant_id <(optional)tenant-id> --client_id <(optional)client-id> --client_secret <(optional)client-secret>
+```
+
+1. Navigate to the support_bundle directory
+2. Setup virtual environment to run the script in
+3. activate the virutal environment
```
Comment (jmolnar-zscaler) [inline scripts/support_bundle/README.md:16]: just a typo on virtual

File: scripts/support_bundle/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Support Bundle Generation Script
+
+This script is to be used to collect information on your Cloud Connector Scale Set Deployment Zscaler Customer Support. The script will collect the health and scale metrics published by the Cloud Connectors, the configuration of the Scale Set deployment, and the logs from the Function App. This bundle will be used for debugging purposes to help support customer.
```
Comment (jmolnar-zscaler) [inline scripts/support_bundle/README.md:3]: are there any special GET privileges needed in order to connect and procure all these logs that are worth mentioning? Or should it just be "run it with the same account/Service principal you used to deploy the env and you should be good"?

**Review by jmolnar-zscaler — COMMENTED** [review]
comments left. no code concerns. just general functionality questions

File: scripts/support_bundle/README.md (hunk 2 above)
Comment (rhartnett-zscaler) [reply scripts/support_bundle/README.md:3]: I have an action item to follow up on permissions needed for this, I will provide that update in a different PR. Thanks for calling this out.

File: scripts/support_bundle/README.md (hunk 1 above)
Comment (rhartnett-zscaler) [reply scripts/support_bundle/README.md:16]: Will adress this in upcoming PR with permission information

**Pull request merged**